src/lib/navigation/navigate.ts

```diff
@@ -160,9 +167,9 @@ export function dismissModal() {
   }
 }
 
-export function goBack() {
+export function goBack(backProps?: BackProps) {
```

Comment:
something like this? 🤔 
```suggestion
export function goBack(fromScreen?: string) {
```

Reply:
Yes, this option also makes sense. Just using the object may come in handy in the future, when we want to pass not only `fromScreen` prop (for example, some useful information from the screen or form data 🤔)